### DISCUSSION TITLE
Modules Security

### DISCUSSION THREAD

**Discussion by jvirtuel** [nWidart/laravel-modules#1385] [Q&A] [answered]
Hello,
I am building a modular laravel app, 
I want to have some freelancers develop modules for me without giving them access the the main core application
in addition, even though i am going to verify each module meticulously, i was wondering how can i secure what module can or can not do first in relation to the core and main database as well as with other modules..
The idea is to make sure they do not temper with my app, stile information or inject anything that is not authorized..
Any recommendations would be nice.
Thanks in advance for your help !

**Comment by laradude** [comment, marked as answer]
I would seed the DB with fake data and have the freelancers build the modules as a composer package. Check out this page https://docs.laravelmodules.com/v9/publishing-modules
